### PR TITLE
AIR-859 : Fix security vulns detected by Trivy in Calico-node

### PR DESCRIPTION
**Pull request by Rohitrajak1807**
- Fix dockerfile issues

### CODE REVIEW

File: filesystem/licenses/Mozilla-public
Comment (cruizen) [inline filesystem/licenses/Mozilla-public:1]: Empty files for licenses?

File: filesystem/etc/calico/felix.cfg
Comment (cruizen) [inline filesystem/etc/calico/felix.cfg:1]: Why did we have to remove this file?

File: filesystem/etc/nsswitch.conf
Comment (cruizen) [inline filesystem/etc/nsswitch.conf:1]: And why did we remove this file?

File: filesystem/licenses/Mozilla-public
Comment (Rohitrajak1807) [reply filesystem/licenses/Mozilla-public:1]: filesystem is a geenrated rootfs

File: filesystem/etc/calico/felix.cfg
Comment (Rohitrajak1807) [reply filesystem/etc/calico/felix.cfg:1]: filesystem is a geenrated rootfs

Comment (Rohitrajak1807) [reply filesystem/etc/calico/felix.cfg:1]: also, the diff is misleading... the file exists and needs a 7xy permission for the Makefile, so the the cleanup succeeds. The contents are unaltered

File: filesystem/etc/nsswitch.conf
Comment (Rohitrajak1807) [reply filesystem/etc/nsswitch.conf:1]: check comment above

File: filesystem/licenses/Mozilla-public
Comment (Rohitrajak1807) [reply filesystem/licenses/Mozilla-public:1]: check comment above

**Review by cruizen — APPROVED** [review]

**Pull request merged**